### DISCUSSION TITLE
Testing standard headers

### DISCUSSION THREAD

**Discussion by acdha** [schemathesis/schemathesis#2160] [Q&A]
For internet-facing services, it would be useful to say that every endpoint has common HTTP headers added to the test cases – for example, `Host`, `X-Forwarded-For` / `X-Forwarded-Proto`, `Range`, `Referer`, `User-Agent`, etc. are all commonly used by application code in ways which may not be included in a schema because they're either undocumented (XFF) or simply assumed as standard HTTP features. It would be useful to have a built-in way to generate test cases which mutate those values even if they're not explicitly listed.

**Comment by Stranger6667** [comment]
Cool idea! I’d add it under the ‘—contrib’ namespace CLI option that will accept a comma-separated list of headers
